.gitignore

```diff
@@ -1,2 +1,21 @@
 .vscode
 Documentation-GENERATED-temp
+
+.Build
```

Comment:
Suggest to move modifications to .gitignore, packages for require-dev in composer.json, Documentation etc. to separate PRs. This might also make this patch easier to review and get merged faster. The changes are for building locally and do not really have anything to do with updating to v11, right?

Reply:
Your're right. This would be better. But I'll take it for now - if I understand the Documentation ;-)

Reply:
The `.gitignore` stuff is also related to the local dev environment, I planned to provide.